config.json

```diff
@@ -240,10 +246,10 @@
             "MsBuildLogging":"/flp:v=normal;LogFile=build-packages.log"
           }
         },
-        "portableLinux":{
+        "portable":{
           "description": "Make the build-native script generate binaries that are portable over glibc based Linux distros.",
           "settings": {
-            "RuntimeOS":"linux"
```

Comment:
We will need to do some work to ensure the RuntimeOS is correctly set when we are in portable mode as well otherwise we will not be pulling in the correct set of native binaries from external. This currently works on linux because we were setting RuntimeOS and it got picked up here https://github.com/dotnet/corefx/blob/master/external/dir.props#L5. We either need to continue to pass the right RuntimeOS or need to take into account the portable flag in the external props.